tests/get_state_dns_test.go

```diff
@@ -29,7 +29,7 @@ func TestGetStateDNS(t *testing.T) {
 	}
 	//receiving the output for every alias and storing the results into a map
 	received := make(map[string][]interface{})
-	iprecString := []string{}
```

Comment:
@pedroldk Proposing this change as "code cleanup" is misleading because it might introduce bugs. The original code initialize the slice to a zero element array, while the new one initialize to `nil`.

The difference might not matter in this code, but advertising this change as just "code cleanup" is wrong.